frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalettePreview/CommandPreview.tsx

```diff
@@ -0,0 +1,47 @@
+import type { StaticImageData } from 'next/image'
+import type { FC } from 'react'
+import copyLink from './assets/copy-link.mp4'
+import showAllFields from './assets/show-all-fields.png'
+import showKeyOnly from './assets/show-key-only.png'
+import showTableName from './assets/show-table-name.png'
+import tidyUp from './assets/tidy-up.mp4'
+import zoomToFit from './assets/zoom-to-fit.mp4'
+import styles from './CommandPalettePreview.module.css'
+
+type Props = {
+  commandName: string
+}
+
+const COMMAND_VIDEO_SOURCE: Record<string, string> = {
+  'copy link': copyLink,
+  'Zoom to Fit': zoomToFit,
+  'Tidy Up': tidyUp,
+}
+
+const COMMAND_IMAGE_SOURCE: Record<string, string | StaticImageData> = {
+  'Show All Fields': showAllFields,
+  'Show Key Only': showKeyOnly,
+  'Show Table Name': showTableName,
+}
+
+const getImageSrc = (imageSrc: string | StaticImageData) =>
+  typeof imageSrc === 'string' ? imageSrc : imageSrc.src
+
+export const CommandPreview: FC<Props> = ({ commandName }) => {
+  return (
+    <div className={styles.container}>
+      {COMMAND_VIDEO_SOURCE[commandName] && (
+        <video muted autoPlay className={styles.video} key={commandName}>
+          <source src={COMMAND_VIDEO_SOURCE[commandName]} type="video/mp4" />
+        </video>
+      )}
+      {COMMAND_IMAGE_SOURCE[commandName] && (
+        <img
+          src={getImageSrc(COMMAND_IMAGE_SOURCE[commandName])}
+          className={styles.image}
+          alt=""
```

Comment:
I think these images are basically for decorations. It may help understanding, but the option names are good enough to understand the command contents. Therefore I didn't set additional `alt` texts for them.